src/data/faq_de.json

```diff
@@ -87,6 +87,126 @@
+                        ]
+                    },
+                    {
+                        "title": "Wie handele ich, wenn mein Schnelltest-Ergebnis positiv ist?",
+                        "anchor": "rat_what_if_positive_result",
+                        "active": true,
+                        "textblock": [
+                            "In Fall eines positiven Testergebnisses sollten Sie Ihr Testergebnis teilen und umgehend einen PCR-Test machen, um das Ergebnis zu bestätigen oder zu widerlegen. Bei einem positiven PCR-Test sollten Sie das Testergebnis erneut teilen, um Kontakte zu warnen, die Sie möglicherweise zwischen den beiden Tests getroffen haben. Voraussetzung dabei ist, dass die Kontaktpersonen ebenfalls die Corona-Warn-App nutzen und die Risikoberechnung ein hohes Risiko bestimmt, basierend auf Dauer und Abstand des Kontakts. Sollte der PCR-Test negativ sein, haben Sie nicht die Möglichkeit, Ihren positiven Schnelltest und die Warnung zurückzuziehen."
+                        ]
+                    },
+                    {
+                        "title": "Wenn ich eine hohe Risikobegegnung in der App angezeigt bekomme, d.h. eine rote Karte/Kachel, sollte ich dann einen Schnelltest oder PCR-Test machen?",
+                        "anchor": "rat_red_card_which_test",
+                        "active": true,
+                        "textblock": [
+                            "Wenn die Corona-Warn-App Ihnen eine Begegnung mit erhöhtem Risiko (rote Kachel) anzeigt, sollten Sie einen PCR-Test machen und sich in Quarantäne begeben."
+                        ]
+                    },
+                    {
+                        "title": "Werden Schnelltest und PCR-Test in der App unterschiedlich behandelt?",
+                        "anchor": "rat_same_in_cwa",
+                        "active": true,
+                        "textblock": [
+                            "Der Schnelltest wird von der Corona-Warn-App gleichberechtigt zum PCR-Test behandelt. Wenn Sie sich bei einem Corona-Warn-App Partner testen lassen, erhalten Sie Ihr Testergebnis per QR-Code, den sie mit der Corona-Warn-App scannen können. Ein negatives Testergebnis wird dann für 48 Stunden in der Corona-Warn-App angezeigt.",
+                            "Im Falle eines positiven Testergebnisses, können Sie Ihre Mitmenschen innerhalb kürzester Zeit warnen, indem Sie Ihr Ergebnis – wie bei einem PCR-Test auch – teilen und so auf den Server der Corona-Warn-App hochladen. Das positive Testergebnis wird Ihnen so lange in der App angezeigt, bis Sie es teilen. Wenn Sie zuvor über die Event-Registrierung bei einem Event oder an einem Ort eingecheckt waren, können Sie das positive Testergebnis, gemeinsam mit Ihrem Check-in teilen."

```

Comment:
No , before gemeinsam in the last sentence

Reply:
thanks